src/Tribe/Admin/Organizer_Chooser_Meta_Box.php

```diff
@@ -50,7 +54,18 @@ protected function get_event( $event = null ) {
 	public function render() {
 		$this->render_dropdowns();
 		$this->render_add_organizer_button();
-		include $this->tribe->pluginPath . 'src/admin-views/new-organizer-meta-section.php';
+
+		if ( class_exists( 'Tribe__Events__Community__Main' ) ) {
```

Comment:
If someone held me at NERF-gun point and asked if I'd prefer a filter or baking in awareness to Events Community, I'd probably say "filter". Luckily, nobody is pointing a NERF gun at me suction cup-tipped or otherwise...so I'm safe, guys. Don't worry. But, yeah. +1 for filter.